manimlib/camera/camera.py

```diff
@@ -51,6 +64,15 @@ class Camera(object):
     }
 
     def __init__(self, background=None, **kwargs):
+        """Initialises the Camera.
```

Comment:
Init documentation goes into the class docstring, using "Parameters". (Just move the Parameters section to the class docstring.)